node-operators/networks/run-a-node/flags.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Flags
+description: Helpful flags for running a full parachain node on Moonbeam
+---
+
+# Helpful Flags for Running a Node on Moonbeam
+
+![Full Node Moonbeam Banner](/images/node-operators/networks/run-a-node/flags/flags-banner.png)
+
+## Introduction {: #introduction }
+
+When spinning up your own Moonbeam node, there are some required and optional flags that can be used.
+
+This guide will cover some of the most common flags and show you how to access all of the available flags.
+
+## Common Flags {: #common-flags }
+
+- **`--validator`** - enables validator mode for collator candidates and, if eligible, allows the node to actively participate in block production
+- **`--port`** - specifies the peer-to-peer protocol TCP port. The default port for parachains is `{{ networks.parachain.p2p }}` and `{{ networks.relay_chain.p2p }}` for the embedded relay chain
+- **`--rpc-port`**  - specifies the HTTP RPC server TCP port. The default port for parachains is `{{ networks.parachain.rpc }}`  and `{{ networks.relay_chain.rpc }}` for the embedded relay chain
+- **`--ws-port`** - specifies the WebSockets RPC server TCP port. The default port for parachains is `{{ networks.parachain.ws }}`  and `{{ networks.relay_chain.ws }}` for the embedded relay chain
+- **`--execution`** - specifies the execution strategy that should be used by all execution contexts. The Substrate runtime is compiled into a native executable and a WebAssembly (wasm) binary. The available options are:
+    - **`native`** - only execute with the native build
+    - **`wasm`** - only execute with the wasm build
```

Comment:
I would talk a bit more about what this means, this uses the WASM binary that is stored on-chain, while the `native` option uses the one local from the node.